### PR TITLE
kubernetes: 2.2.2-1.13.5 docs

### PR DESCRIPTION
**Pull request by jimmidyson**
## Description
Documentation for dcos-kubernetes 2.2.2-1.13.5.

## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/kubernetes/2.2.2-1.13.5/limitations/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Configuration
+
+* Updating the value of the `kubernetes.authorization_mode` option after the creation of a cluster is not supported.
+* Updating the value of the `kubernetes.high_availability` option to `false` after the creation of a cluster is not supported.
+* Updating the value of the `service.region` option after the creation of a cluster is not supported.
+
+# DC/OS Integration
+
+* There is no integration between the DC/OS UI and the Kubernetes Dashboard.
+* Mesos roles are not supported.
```
Comment (jpe442) [inline pages/services/kubernetes/2.2.2-1.13.5/limitations/index.md]: We support starting in 13.4 ya?

**Comment by MatApple** [comment]
After we address the request from @jpe442 this LGTM.

File: pages/services/kubernetes/2.2.2-1.13.5/limitations/index.md (hunk 1 above)
Comment (jimmidyson) [reply pages/services/kubernetes/2.2.2-1.13.5/limitations/index.md]: Removed - good catch.

**Comment by jimmidyson** [comment]
@jpe442 Please re-review and merge if you're happy :) Then let's get this staged.

**Comment by jpe442** [comment]
🚀🚀🚀

**Pull request merged**